source/_posts/Rails-Starter-Template.md

```diff
@@ -0,0 +1,209 @@
+---
```

Comment:
Link to previous Content - in content there should be link with previous content

Reply:
The content is not related to any previous content, hence not added link. Have added link to Nayan website.